### DISCUSSION TITLE
Recommendation of hardware.

### DISCUSSION THREAD

**Discussion by janiversen** [frenck/home-assistant-config#68] [General]
Hi @frenck 

I looked for a the least disturbing way to contact you, and hope I selected the right one.

I have looked at this repo for a while, and that is how I want my home to be 😁 

I am about to start a major reform of our flat (tear down walls/floors, add underfloor heating etc.) and of course I want it all to be run by HA. Apart from creating a nice modern flat, I want the automation part to be something to show. Here in Spain automation is still very new and very expensive. I want to help change that, by e.g. showing how HA can help you automate in a good but relatively cheap way.

So may I politely ask you, if you were to start over with the products currently available, what products would you choose for contacts/contacts/blinds ? Being a modbus fan, I would like that, but it is not a good choice.

Thanks in advance for responding.

Rgds
Jan I.

Ps. If you prefer not to "advertise" products (which I can understand), send me a message in discord "janGranada".

**Comment by frenck** [comment]
Hehe, Discord is also always OK for this 👍 

Nevertheless, I dunno actually. I have a mix of quite a lot of different things, mainly because I like playing and testing a lot.

At this point, I mainly use Zigbee, Z-Wave & ESPHome.

Z-Wave is nice and stable again with the Z-Wave JS project.

Zigbee... it is cheap, but quality differs in brands. For example, Hue stuff works fine, Ikea is OK but Xiaomi is the cheapest stuff but can be flaky. The main problem with Zigbee is that a lot of manufacturers do not follow specs :)

ESPHome, those devices are by far the most stable (for me). It is fun to use/create and local. A decent Wi-Fi network is required though.

That all said, I might not be the best example. I work on Home Assistant a lot, which also means my home is kinda the playground that is always in a state of any of the following:
- Needs to be fixed
- Half done, got distracted into coding in HA again
- Torn apart for another test or reproduction of a bug

😬

**Comment by niro1987** [comment]
From my experience;

- Smart lights are only as smart as the switch they are connected to.
It doesn't make much sense to hook up a smart light to a dumb switch that just has te be in the 'on' state all the time.
If you can't make the light smart, a smart switch will still do the trick.
- Don't try to change the behaviour of the user - they won't.
- Automations should add value, not replace anything.

**Reply by frenck** [reply]
> Smart lights are only as smart as the switch they are connected to.

☝️ That is true. I'm personally a big violator of that rules, simply because I like dimmable/color temperature lights a lot as they bring more natural lighting into our house. Thus no physical working light switches anymore.

> Automations should add value, not replace anything.

💯  I like to add to that: Should be invisible and barely noticeable.

**Reply by janiversen** [reply]
no physical light switches, what happens when your network is down? candlelight

**Reply by frenck** [reply]
Local control, so I can handle internet outage. If my home network goes down 🤷  early bed.

**Reply by niro1987** [reply]
Personally, I've made the switches smart by hooking them up to an ESP. Sonoff Mini's mostly - has Wifi, can be connected directly to the mains - there are several other devices out there that do the same thing. The ESP is just there to pick up the trigger for HA to toggle the light. But if you tear down the walls anyway - I would go for zigbee buttons instead - saves you a lot of hassle with the wires. The uspide of having the normal switches is that visitors kind of expect them to be there.

**Reply by janiversen** [reply]
I looked at zigbee at liked the hardware, but I read that you need to be careful because a lot of the devices use encrypted traffic and are therefore difficult to integrate in e.g. HA.

**Comment by janiversen** [comment]
I have been playing with an old danish idea. Make all your switches 2 12v buttons and then installs relays with dimmers in your distribution panels. I do not like the BIG 220 switches a small plate with 2 buttons in the door panel is much more discrete.

But I have not been able to find such relays (with dimmer, digital + manual) for a house only for factories etc.

**Reply by frenck** [reply]
Shelly dimmers :)

**Reply by janiversen** [reply]
they are really cheap, but I like the technical specification “communication: wifi”. 

But since you mention them I am sure “wifi” can be integrated i. HA 😁
